Recipes.Tests/WebApiTelemetryTests.cs

```diff
@@ -138,16 +140,36 @@ public void TearDown()
             }
 
             telemetryEvents.Single()
-                           .IsIncomingRequest()
-                           .Should()
-                           .BeTrue();
-            telemetryEvents.Single()
                            .CallerIpAddress()
                            .Should()
                            .Be("123.123.123.123");
         }
 
         [Test]
+        public async Task Telemetry_events_based_on_HTTP_request_are_marked_as_incoming()
+        {
+            HttpResponseMessage response = null;
+
+            using (Log.With<Telemetry>(t => t.WithPropertiesBasedOn(response)).Enter(() => { }))
+            {

```

Comment:
This says `WithPropertiesBasedOn(response)` which seems to be different from what the test name says.